ironfish-rust/src/frost_utils/split_spender_key.rs

```diff
@@ -89,21 +93,11 @@ mod test {
         let trusted_dealer_key_packages =
             split_spender_key(&sapling_key, 5, identities).expect("spender key split failed");
 
-        assert_eq!(
-            trusted_dealer_key_packages.key_packages.len(),
-            10,
-            "should have 10 key packages"
-        );
```

Comment:
Why was this assertion removed?

Reply:
Added it back, not sure how it got removed. must have made a mistake